api/src/main/java/fr/catcore/server/translations/api/resource/language/ServerLanguageManager.java

```diff
@@ -131,6 +106,12 @@ private ServerLanguage createLanguage(ServerLanguageDefinition definition) {
         ServerLanguage language = new ServerLanguage(definition);
         this.languages.put(definition.getCode(), language);
 
+        // we add all the default translation keys to other languages
```

Comment:
This should not apply to vanilla keys.

Reply:
I might've misunderstood the previous code, but I removed this since I made the future calls to `put` overwrite these values. What might happen if this sets the vanilla keys too?

Reply:
It will set them to the english ones :/

Reply:
This was happening before because internally the language map used `putIfAbsent` - it's just using `put` now, so the later calls will have priority